test/integration/imagesigning_test.go

```diff
@@ -194,7 +196,6 @@ func TestImageRemoveSignature(t *testing.T) {
 }
 
 func testSetupImageSignatureTest(t *testing.T, userName string) (adminClient *client.Client, userClient *client.Client, image *imageapi.Image) {
-	testutil.RequireEtcd(t)
```

Comment:
Hmmm... you're not consistent. In `test/integration/imageimporter_test.go` you do this inside `testImageStreamImportWithPath` helper method, but here you moved those to actual test methods.
In general I'm fine with this approach, but make it consistent everywhere.